test/views/ReportingFrequencyViewSpec.scala

```diff
@@ -38,48 +37,88 @@ class ReportingFrequencyViewSpec extends TestSupport {
 
   val govGuidanceUrl = "https://www.gov.uk/guidance/find-software-thats-compatible-with-making-tax-digital-for-income-tax"
 
+  def nextUpdatesUrl(isAgent: Boolean): String =
+    if(isAgent) controllers.routes.NextUpdatesController.showAgent().url
+    else controllers.routes.NextUpdatesController.show().url
+
   def bullet(i: Int): String = s"#main-content > div > div > div > ul > li:nth-child($i) > a"
 
+
   object Selectors {
     val h1 = "reporting-frequency-heading"
     val h2 = "manage-reporting-frequency-heading"
     val p1 = "change-reporting-frequency"
     val p2 = "what-you-can-do"
-    val latencyDetailsDropdownPara1 = "separately-choose-to-opt-out"
```

Comment:
this section appears to be completely replaced which I don't think is quite right. 

In the prototype:

Page Version **Quarterly reporting - with latent business**

shows both sections dropdown and new different-obligations content.